Frends.SFTP.ListFiles/Frends.SFTP.ListFiles/Frends.SFTP.ListFiles.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.ComponentModel;
+using Renci.SshNet;
+using System.Net.Sockets;
+using Renci.SshNet.Common;
+using System.Text.RegularExpressions;
+using Frends.SFTP.ListFiles.Definitions;
+
+#pragma warning disable 1591
+
+namespace Frends.SFTP.ListFiles
+{
+    public class SFTP
+    {
+        /// <summary>
+        /// Reads a file with SFTP connection.
+        /// Documentation: https://github.com/FrendsPlatform/Frends.SFTP.ReadFile
+        /// </summary>
+        /// <param name="connection">Transfer connection parameters</param>
+        /// <param name="options">Source file location</param>
+        /// <param name="cancellationToken"></param>
```

Comment:
I don't think cancellationToken needs to be documented separately, since it is passed by Frends automatically and it doesn't require any user input.

Reply:
Fixed